sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/RecordImpl.java

```diff
@@ -34,15 +35,15 @@ public Key getKey() {
     }
 
     @Override
-    public Iterator<Map.Entry<UUID, Compaction>> passOneIterator() {
-        Iterator<Map.Entry<UUID, Compaction>> result = checkNotNull(_passOneIterator, "Already consumed.");
+    public Iterator<Map.Entry<DeltaClusteringKey, Compaction>> passOneIterator() {
+        Iterator<Map.Entry<DeltaClusteringKey, Compaction>> result = checkNotNull(_passOneIterator, "Already consumed.");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Already consumed." 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)